src/main/java/org/example/TaskManager.java

```diff
@@ -0,0 +1,133 @@
+import java.util.Scanner;
+
+public class TaskManager {
+    static File file = new File("tasks.csv");
+    static String[][] tasks;
+    static final String[] OPTIONS = {"add", "remove", "list", "exit"};
+    public static void showMainMenu() {
+        System.out.println(ConsoleColors.BLUE + "\nPlease select an option: " + ConsoleColors.RESET);
+        for (String option : OPTIONS) {
+            System.out.println("-> " + option);
+        }
+        menuButtons();
+    }
+
+    public static void menuButtons() {
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
+
+        switch (input) {
+            case "add":
+                add();
+                break;
+            case "remove":
+                remove();
+                break;
+            case "list":
+                list();
+                break;
+            case "exit":
+                exit();
+                break;
+            default:
+                System.out.println(ConsoleColors.RED + "Please select a correct option." + ConsoleColors.RESET);
+                menuButtons();
+                break;
+        }
+    }
+
+    public static void add() {
+        Scanner scanner = new Scanner(System.in);
+        tasks = Arrays.copyOf(tasks, tasks.length + 1);
+        System.out.println("Please enter the name of the task you would like to add:");
+        String name = scanner.nextLine();
+        System.out.println("Please enter the date (format YYYY-MM-DD):");
+        String date = scanner.nextLine();
+        System.out.println("Please enter if the task is done or not (true/false):");
+        String bool = scanner.nextLine();
+        tasks[tasks.length - 1] = new String[] {name, date, bool};
+        showMainMenu();
+    }
+
+    public static void remove() {
+        System.out.println("Please, enter the number of the task you would like to remove:");
+        int number = -1;
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String input = scanner.nextLine();
+            try{
+                number = Integer.parseInt(input);
+            } catch (NumberFormatException e) {
+            }
+            if (number >= 0 && number < tasks.length-1) {
+                break;
+            } else {
+                System.out.println(ConsoleColors.RED + "Please, enter a number between 0 and " + (tasks.length - 1) + "." + ConsoleColors.RESET);
+            }
+        }
+        tasks = ArrayUtils.remove(tasks, number);
+        showMainMenu();
+    }
+
+    public static void list() {
+        for (int i = 0; i < tasks.length; i++) {
+            for (int j =0 ; j < 3; j++) {
+                if (j < 2) {
+                    System.out.print(ConsoleColors.RESET + tasks[i][j]+ ", ");
+                } else {
+                    System.out.print(ConsoleColors.RESET + tasks[i][j]);
+                }
+            }
+            System.out.println();
+        }
+        showMainMenu();
```

Comment:
CLEAN-CODE: lepsi struktura.

```suggestion
 private static void list() {
        System.out.println(ConsoleColors.PURPLE + "Id:[Note, Date, IsImportant ]");
        System.out.println(ConsoleColors.RESET + "-----------------");
        for (int i= 0; i < tasks.length; i++) {
            System.out.println( i + ":"  + Arrays.toString(tasks[i]));
        }
    }
```

Reply:
opraveno